src/core/job.c

```diff
@@ -1393,6 +1393,9 @@ void job_shutdown_magic(Job *j) {
         if (!unit_has_name(j->unit, SPECIAL_SHUTDOWN_TARGET))
                 return;
 
+        /* This is the very beginning of the shutdown phase, so take the timestamp here */
+        dual_timestamp_now(j->unit->manager->timestamps + MANAGER_TIMESTAMP_SHUTDOWN_START);
```

Comment:
BTW, `j->unit->manager` is superfluous, since `Job` struct already takes a reference to `Manager`. So switch to `j->manager` here and around?

(Maybe add `assert(j->manager)` while at it.